internal/campaign/campaign.go

```diff
@@ -38,11 +38,19 @@ type Campaign struct {
 	PrBody  string
 }
 
-func OpenCampaign() (*Campaign, error) {
+type CampaignOptions struct {
```

Comment:
A bit of an overkill right now, but the idea is to use a data structure describing how to manipulate campaigns.
The repo file is the first of the list, but it could contain some useful extra parameters later.

Reply:
Maybe apply YAGNI and don't make this change until it's needed?